### PR TITLE
Vote-2235: Fixes for Axe Tests

### PR DESCRIPTION
**Pull request by clmedders**
## Jira ticket

[Vote-2235](https://cm-jira.usa.gov/browse/VOTE-2235)

## Description

Addressing 2 fails that were happening in the Cypress Axe tests 

- Image needing alt text:  Now that we are setting alt text on an image as optional this will set up a conditional to check that if alt text is present render the image if not add the attribute of `role="presentation"` to allow the component to pass. [See documentation for more info](https://dequeuniversity.com/rules/axe/4.9/image-alt?application=axeAPI) 
- content needed a wrapper: With the new Voter guide cards it was living outside of main and needed to have a landmark element. [See documentation for more info](https://dequeuniversity.com/rules/axe/4.9/region?application=axeAPI)  

### Post-deploy steps

1. run `lando retune`
2. cd into `votegov` theme and run `npm run build`

### QA/Testing instructions

1. log in as a super user and upload an image and do not add any alt text.  Inspect the image and check to see if the attribute of `role="presentation"`and then add an image with alt text and check that the role is removed and the alt text is being added.
2. cd into `testing` and run `npm run cy:open` and select the Axe test and check that all tests outside of the register to vote are passing (this is getting addressed in another ticket)

## Checklist for the Developer

- [x] A link to the JIRA ticket has been included above.
- [x] No merge conflicts exist with the target branch.
- [x] Automated tests have passed on this PR.
- [x] A reviewer has been designated.
- [x] Deployment and testing steps have been documented above, if applicable.

## Checklist for the Peer Reviewers

- [ ] The file changes are relevant to the task objective.
- [ ] Code is readable and includes appropriate commenting.
- [ ] Code standards and best practices are followed.
- [ ] QA/Test steps were successfully completed, if applicable.
- [ ] Applicable logs are free of errors.

### CODE REVIEW

File: web/themes/custom/votegov/templates/layout/page--node--voter-guide.html.twig [hunk 1]
```diff
@@ -39,6 +39,8 @@
       </div>
     </div>
   </main>
+  <div role="region" aria-labelledby="voter-guide-cards">
```
Comment (clmedders) [inline web/themes/custom/votegov/templates/layout/page--node--voter-guide.html.twig:42]: added this to address the error of this block not having a landmark

Comment (rayestrada) [reply web/themes/custom/votegov/templates/layout/page--node--voter-guide.html.twig:42]: I don't see the aria-labeledby related to any id of a header in the component.

Comment (rayestrada) [reply web/themes/custom/votegov/templates/layout/page--node--voter-guide.html.twig:42]: Can we set it to an aria-label of value `{ 'Know your voting rights' | t '}`?

**Review by rayestrada — CHANGES_REQUESTED** [review]

**Comment by clmedders** [comment]
updated the label to match heading!  i think i read the documentation wrong but see now what you are saying

File: web/themes/custom/votegov/templates/layout/page--node--voter-guide.html.twig [hunk 2]
```diff
@@ -39,7 +39,7 @@
       </div>
     </div>
   </main>
-  <div role="region" aria-labelledby="voter-guide-cards">
+  <div role="region" aria-labelledby="{{ 'Know your voting rights' | t }}">
```
Comment (rayestrada) [inline web/themes/custom/votegov/templates/layout/page--node--voter-guide.html.twig:42]: this needs to be a `aria-label` attribute. `aria-lablledby` expects an id name.

**Review by rayestrada — CHANGES_REQUESTED** [review]

File: web/themes/custom/votegov/templates/layout/page--node--voter-guide.html.twig (hunk 2 above)
Comment (clmedders) [reply web/themes/custom/votegov/templates/layout/page--node--voter-guide.html.twig]: updated to `aria-label` did some more reading of the docs on MDN and think i have a better understanding now of the use between the 2

**Review by rayestrada — APPROVED** [review]

**Pull request merged**